examples/tutorials/ctc_forced_aligment_api_tutorial.py

```diff
@@ -0,0 +1,683 @@
+"""
+CTC forced alignment API tutorial
+=================================
+
+**Author**: `Xiaohui Zhang <xiaohuizhang@meta.com>`__
+
+(Thanks to `Vineel Pratap <vineelkpratap@meta.com>`__ and `Zhaoheng
+Ni <zni@meta.com>`__ for working on the forced aligner API, and `Moto
+Hira <moto@meta.com>`__ for providing alignment visualization
```

Comment:
Can we put the acknowledgment at the bottom? Or can we just mention that this tutorial was brought with the help of those people?

Tutorials are for readers and for the readers, this is not really information that they want to see first.